robottelo/cli/hostcollection.py

```diff
@@ -35,25 +37,42 @@ class HostCollection(Base):
     command_base = 'host-collection'
 
     @classmethod
+    def transform_ids(cls, options=None):
+        """Workaround host unification feature not being completed to use host
+        id instead content host uuid
```

Comment:
Is this something permanent or temporary and we should revisit in future? need to track a Bz for this or open an issue?